Data/Sentinel1_processing_guide.qmd

```diff
@@ -0,0 +1,63 @@
+---
+title: "Derived products & Processing options "
+format:
+  html:
+    code-fold: true
+jupyter: python3
+---
+
+Sentinel-1 data can be accessed and processed in different ways within Copernicus Data Space Ecosystem. Below we have compiled an overview of all the options to help you decide which one to use.
+
+## [Sentinel-1 RTC](https://documentation.dataspace.copernicus.eu/Data/Additional.html#sentinel-1-rtc) 
+[Sentinel-1 RTC](https://documentation.dataspace.copernicus.eu/Data/Additional.html#sentinel-1-rtc) (Radiometric Terrain Correction) SAR Backscatter is a product processed from Sentinel-1 GRD data and compliant with [CEOS Analysis Ready Data for Land (CARD4L) specifications](https://ceos-dev.ceos.org/ard/) for Normalised Radar Backscatter (NRB) products. Orthorectification is based on Copernicus DEM and no speckle filtering is applied. ([Additional product information](https://sentinels.copernicus.eu/web/sentinel/sentinel-1-ard-normalised-radar-backscatter-nrb-product))
```

Comment:
Please add `{target="_blank"}` behind links to external websites. (Point 2 in [Basic guidelines document](https://github.com/eu-cdse/documentation/blob/publish/EU-CDSE_Documentation_Guidelines_v0.pdf))